.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
       - run:
           name: Install flake8 & mypy
-          command: sudo python -m pip install flake8 mypy
+          command: sudo python -m pip install flake8==3.8.1 mypy
```

Comment:
Why is this pinned? I thought we had decided to not pin dev requirements (as having them updated automaticall is too noisy)?
I also can't imagine the version of `flake8` to matter so much.

Reply:
I other project I manage last flake8 upgrade introduce many about 10 errors. Erors not connected with PR, but from upgrade of flake8.

also pre commit has pinned flake8 
But I do not thing this change is mandatory.

Reply:
I think it's quite nice to sometimes get updates. I'm sure we otherwise just forget. If those errors are easy, why not fix them in the PR. If they are hard to fix, the PR can still be merged, even if flake fails, and fixed (or ignored) in another PR/commit.

I thought we had discussed pinning dev-dependencies and decided against. So I don't like adding this now, without thinking further about how to handle this later. And then if we do it, we should probably do it for all. But again, really not a fan. If static analysis gets better, we should benefit from it.

Reply:
If we're gonna pin, let's do it in dev-dependencies. But I'm okay leaving it unpinned, for the reason yannickjadoul mentions.